releaseNotes/Microsoft.Featuremanagement.md

```diff
@@ -4,14 +4,31 @@
 # Microsoft.FeatureManagement.AspNetCore
 [Source code ][source_code_web] | [Package (NuGet)][package_web] | [Samples][samples_web] | [Product documentation][docs]
 
+## 3.1.0 - November 23, 2023
+
+### Enhancements
+
+* `FeatureManager` and `ConfigurationFeatureDefinitionProvider` are now public.
```

Comment:
I thought there was an issue we can reference. Do we have one?

Reply:
Added